README.md

```diff
@@ -1,14 +1,184 @@
+      case 'S': return new Card(rank, Suit.SPADES);
+      default: throw new JsonDataException("unknown suit: " + card);
+    }
+  }
+}
+```
+
+Register the type adapter with the `Moshi.Builder` and we’re good to go.
+
+```java
+Moshi moshi = new Moshi.Builder()
+    .add(new CardAdapter())
+    .build();
+```
+
+Voila:
+
+```json
+{
+  "hidden_card": "6S",
+  "visible_cards": [
+    "4C",
+    "AH"
+  ]
+}
+```
+
+### Fails Gracefully
+
+Automatic databinding almost feels like magic. But unlike the black magic that typically accompanies
+reflection, Moshi is designed to help you out when things go wrong.
+
+```
+JsonDataException: Expected one of [CLUBS, DIAMONDS, HEARTS, SPADES] but was ANCHOR at path $.visible_cards[2].suit
+  at com.squareup.moshi.JsonAdapters$11.fromJson(JsonAdapters.java:188)
+  at com.squareup.moshi.JsonAdapters$11.fromJson(JsonAdapters.java:180)
+	...
+```
+
+Moshi always throws a standard `java.io.IOException` if there is an error reading the JSON document,
+or if it is malformed. It throws a `JsonDataException` if the JSON document is well-formed, but
+doesn’t match the expected format.
+
+### Built on Okio
+
+Moshi uses [Okio][okio] for simple and powerful I/O. It’s a fine complement to [OkHttp][okhttp],
+which can share buffer segments for maximum efficiency.
+
+### Borrows from Gson
+
+Moshi uses the same streaming and binding mechanisms as [Gson][gson]. If you’re a Gson user you’ll
+find Moshi works similarly. If you try Moshi and don’t love it, you can even migrate to Gson without
+much violence!
+
+But the two libraries have a few important differences:
+
+ * **Moshi has fewer built-in type adapters.** For example, you need to configure your own date
+   adapter. Most binding libraries will encode whatever you throw at them. Moshi refuses to serialize
+   platform types (`java.*`, `javax.*, and `android.*`) without a user-provided type adapter. This is
+   intended to prevent you from accidentally locking yourself to a specific JDK or Android release.
+ * **Moshi is less configurable.** There’s no field naming strategy, versioning, instance creators,
+   or long serialization policy. Instead of naming a field `visibleCards` and using a policy class
+   to convert that to `visible_cards`, Moshi wants you to just name the field `visible_cards` as it
+   appears in the JSON.
+ * **Moshi doesn’t have a `JsonElement` model.** Instead it just uses built-in types like `List` and
+   `Map`.
+ * **No HTML-safe escaping.** Gson encodes `=` as `\u003d` by default so that it can be safely
+   encoded in HTML without additional escaping. Moshi encodes it naturally (as `=`) and assumes that
+   the HTML encoder – if there is one – will do its job.
 
 
 
 Download
 --------
 
-**Moshi is currently under development.** The API is not stable and neither is the feature set. It should not be used.
+**Moshi is currently under development.** The API is not stable and neither is the feature set.
```

Comment:
"not stable" -> "mostly stable"
"neither is" -> "so is"

We're a bit more committed I think to make these claims.